mantidproject/fitbenchmarking
# Added comments and improved structure

#### Description of Work

Fixes #48 
As the title says, added comments, improved variable names and improved structure.
Note: Code still needs to be organised in modules, right now it is too messy and still kind of hard to understand.


#### Testing Instructions

1. Run `example_runScripts.py`  in the mantidpython console.
2. Check if the tables work properly.
